src/components/panel/demoBasicUsage/script.js

```diff
@@ -95,52 +98,56 @@ function PanelDialogCtrl(mdPanelRef) {
 
 
 PanelDialogCtrl.prototype.closeDialog = function() {
+  var $document = this.$document;
   var panelRef = this._mdPanelRef;
 
   panelRef && panelRef.close().then(function() {
-    angular.element(document.querySelector('.demo-dialog-open-button')).focus();
+    angular.element($document[0].querySelector('.demo-dialog-open-button')).focus();
     panelRef.destroy();
   });
 };
 
 
 
 function PanelMenuCtrl(mdPanelRef, $timeout) {
+  var $document = this.$document;
   this._mdPanelRef = mdPanelRef;
   this.favoriteDessert = this.selected.favoriteDessert;
   $timeout(function() {
-    var selected = document.querySelector('.demo-menu-item.selected');
+    var selected = $document[0].querySelector('.demo-menu-item.selected');
     if (selected) {
       angular.element(selected).focus();
     } else {
-      angular.element(document.querySelectorAll('.demo-menu-item')[0]).focus();
+      angular.element($document[0].querySelectorAll('.demo-menu-item')[0]).focus();
     }
   });
 }
 
 
 PanelMenuCtrl.prototype.selectDessert = function(dessert) {
+  var $document = this.$document;
   this.selected.favoriteDessert = dessert;
   this._mdPanelRef && this._mdPanelRef.close().then(function() {
-    angular.element(document.querySelector('.demo-menu-open-button')).focus();
+    angular.element($document[0].querySelector('.demo-menu-open-button')).focus();
   });
 };
 
 
 PanelMenuCtrl.prototype.onKeydown = function($event, dessert) {
+  var $document = this.$document;
```

Comment:
This is broken in the demo. Pressing the up/down arrow buttons causes the whole window to scroll and does not change options in the dropdown.